endpoints/books/controller.py

```diff
@@ -24,38 +28,40 @@
 
 
 class UserBookResource(Resource):
+    @jwt_required
     def post(self):
         try:
+            current_user = get_jwt_identity()
             book = request.get_json()
+            book['user_id'] = current_user
             db.session.add(UserBook(**book))
             db.session.commit()
             return marshal(book, book_list_fields)
 
         except Exception as error:
```

Comment:
Could you handle errors more gracefully? What if the book already exists and the database fails? It should return 422 or something like that.